### PR TITLE
Aws amp integration

### PR DESCRIPTION
**Pull request by linhlam-kc**
Changes log:

- Re-write the instructions for the new integration with AWS Managed Prometheus service.

Note: 
- This doc is pending on new version (1.97.0) to be published.
- This doc is written to use container images and helm chart from Amazon ECR public gallery in honor of the collaboration with AWS.

### CODE REVIEW

**Comment by linhlam-kc** [comment]
Related Issue: https://github.com/kubecost/docs/issues/333

**Comment by harry671003** [comment]
Thank you for adding this nicely written doc. :)
I noticed some minor issues with the arguments passed to the helm command.

**Comment by linhlam-kc** [comment]
> Thank you for adding this nicely written doc. :)
I noticed some minor issues with the arguments passed to the helm command.

Thank you @harry671003 I will make those changes in the next commit, they were my typo when replacing values from my testing notes.

File: aws-amp-integration.md
Comment (linhlam-kc) [reply aws-amp-integration.md:54]: Thanks ! Updating the doc to:
```
The workspace should be created in a few seconds. You can log in to [AWS AMP console](https://console.aws.amazon.com/prometheus/) to retrieve more information. You need to set `$REMOTEWRITEURL` and `$QUERYURL` for using in the integration with Kubecost later as follows:

REMOTEWRITEURL="https://aps-workspaces.us-west-2.amazonaws.com/workspaces/$<AMP_WORKSPACE_ID>/api/v1/remote_write"
QUERYURL="http://localhost:8005/workspaces/$<AMP_WORKSPACE_ID>"
```

File: aws-amp-integration.md
Comment (linhlam-kc) [reply aws-amp-integration.md:129]: Thanks ! Updating the command to:

```bash
helm upgrade -i kubecost \
oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
--namespace kubecost --create-namespace \
-f https://tinyurl.com/kubecost-amazon-eks
-f PATH_TO_THE_LOCAL_DIRECTORY/values-amp.yaml
```

File: aws-amp-integration.md
Comment (linhlam-kc) [reply aws-amp-integration.md:119]: Thanks ! updating the doc

```bash
helm upgrade -i kubecost \
oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
--namespace kubecost --create-namespace \
-f https://tinyurl.com/kubecost-amazon-eks
-f https://tinyurl.com/kubecost-amp
--set global.amp.prometheusServerEndpoint ${QUERYURL}
--set remoteWriteService ${REMOTEWRITEURL}
```

File: aws-amp-integration.md
Comment (linhlam-kc) [reply aws-amp-integration.md:118]: Thanks ! updating the doc

```bash
helm upgrade -i kubecost \
oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
--namespace kubecost --create-namespace \
-f https://tinyurl.com/kubecost-amazon-eks
-f https://tinyurl.com/kubecost-amp
--set global.amp.prometheusServerEndpoint ${QUERYURL}
--set remoteWriteService ${REMOTEWRITEURL}
```

**Comment by linhlam-kc** [comment]
@harry671003  Can you review the latest commit and let me know if there are anything need to be changed/updated to get the doc approved?

@bstuder99 Hi Brett, this doc is for the new AWS integration launched by 09/22. It would be great if you can help to take a look in term of style, format, language etc ... before the launch.

**Comment by bstuder99** [comment]
Okay, I've gone through it and made style edits, LGTM

**Comment by harry671003** [comment]
Thanks again for adding this doc. LGTM

**Comment by linhlam-kc** [comment]
> Thanks again for adding this doc. LGTM

Thank you Harry ! Can you help to approve this PR as reviewer if possible?

cc @dwbrown2 for your visibility and please let me know if you have any additional feedback before I merge the PR (Kubecost-doc) to support AMP integration launch. Thanks

**Review by harry671003 — APPROVED** [review]

**Comment by dwbrown2** [comment]
@linhlam-kc looks good to me! Two questions:

* Would you like someone from core eng to review? cc @biancaburtoiu @AjayTripathy 
* Should we add a link to new blog post when available?

File: aws-amp-integration.md
Comment (dwbrown2) [inline aws-amp-integration.md:147]: @AjayTripathy do you think these should be included given their current role?

**Comment by linhlam-kc** [comment]
> @linhlam-kc looks good to me! Two questions:
> 
> * Would you like someone from core eng to review? cc @biancaburtoiu @AjayTripathy
> * Should we add a link to new blog post when available?
@dwbrown2 

1. It would be great if we have 2nd eye on it. I might be biased with the doc I wrote. I have tested it again from end to end today and it is working fine. It only need 1 minor fix to use 1.97.0 images which I created a PR to fix here https://github.com/kubecost/cost-analyzer-helm-chart/pull/1705 . While waiting for that PR to be merged, using this parameter can ensure that 1.97.0 images are used `--set imageVersion=prod-1.97.0`

2. Yes, I can either wait for the blog post to be released before releasing this doc or release this one first then update it later with link to AWS blog.

**Comment by linhlam-kc** [comment]
This doc is also tested by AWS AMP engineer team so I think we are good to push it out now and update the link to official AWS doc + blog post later.

File: aws-amp-integration.md
Comment (dwbrown2) [inline aws-amp-integration.md:83]: shouldn't this have already happened in previous step?

**Review by dwbrown2 — APPROVED** [review]
I haven't completed each install step, but LGTM to me otherwise. If we post beforehand, proposal is to add blog link later.

File: aws-amp-integration.md
Comment (AjayTripathy) [reply aws-amp-integration.md:147]: No, I don't think they're necessary.

File: aws-amp-integration.md
Comment (linhlam-kc) [reply aws-amp-integration.md:83]: I include it to ensure the command will work even when customer switch between the terminals. If you think that it is unnecessary, I can remove it before merging.

Comment (linhlam-kc) [reply aws-amp-integration.md]: Btw, I removed them to make it align with AWS doc and blog post. Thank you Webb !

Comment (dwbrown2) [reply aws-amp-integration.md]: Feels like it's fine either way, just wanted to call out in case it caused an error

**Pull request merged**
